react_ui/src/components/layout/NavBar.tsx

```diff
@@ -114,7 +87,7 @@ const HeaderNavLink: React.FC<{
     <NavLink
       onClick={onClick}
       to={to}
-      activeClassName="text-mfd-pink-1"
+      activeClassName="fix-me"
```

Comment:
Not sure what our plan is for the top nav but figured I'd leave this capability in here until we have a clear idea of how it'll shake out. Changed the class name as part of removing the M4D-specific code.